modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -652,26 +650,29 @@ namespace {
  * Takes an identifier to a sky browser and adds a rendered copy to it.
  * \param raePosition Position in radius, azimuth, elevation coordinates
  * \param nCopies Number of copies
+ * \param position Position of first copy, which depends on if RAE or Cartesian coordinates
+ * are checked
  */
 [[codegen::luawrap]] void addRenderCopy(std::string identifier,
                                         int numberOfCopies = 1,
-                     glm::vec3 radiusAzimuthElevationPosition = glm::vec3(2.1f, 0.f, 0.f))
+                     glm::vec3 position = glm::vec3(2.1f, 0.f, 0.f))
```

Comment:
the beginning of this line should align with the `(` of the function if it fits, otherwise right-aligned